applications/openshift/api-server/api_server_tls_security_profile/tests/tls_security_profile.fail.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+mkdir -p /tmp/apis/config.openshift.io/v1/apiservers
+
+cat << EOF > /tmp/apis/config.openshift.io/v1/apiservers/cluster
```

Comment:
The tests indicate that the check was not applicable, I am suspicious that you need to actually use this file instead:

```suggestion
cat << EOF > /kubernetes-api-resources/apis/config.openshift.io/v1/apiservers/cluster
```